src/otx/v2/adapters/datumaro/adapter/action_dataset_adapter.py

```diff
@@ -110,86 +108,37 @@ def get_otx_dataset(self) -> DatasetEntity:
+    def get_otx_dataset(self) -> dict[Subset, DatumDataset]:
+        """Get DatasetEntity.
 
-    def get_otx_dataset(self) -> DatasetEntity:
-        """Convert DatumaroDataset to DatasetEntity for Acion Detection."""
-        # Detection use index 0 as a background category
-        for label_entity in self.label_entities:
-            label_entity.id = ID(int(label_entity.id) + 1)
+        Args:
+            datumaro_dataset (dict): A Dictionary that includes subset dataset(DatasetEntity)
 
-        dataset_items: List[DatasetItemEntity] = []
-        for subset, subset_data in self.dataset.items():
+        Returns:
+            DatasetEntity:
+        """
+        for _, subset_data in self.dataset.items():
             for _, datumaro_items in subset_data.subsets().items():
                 for datumaro_item in datumaro_items:
-                    image = self.datum_media_2_otx_media(datumaro_item.media)
-                    shapes: List[Annotation] = []
                     is_empty_frame = False
                     for annotation in datumaro_item.annotations:
-                        if isinstance(annotation, DatumBbox):
-                            if self.label_entities[annotation.label].name == self.EMPTY_FRAME_LABEL_NAME:
-                                is_empty_frame = True
-                                shapes.append(self._get_label_entity(annotation))
-                            else:
-                                shapes.append(self._get_original_bbox_entity(annotation))
-
-                    video_name, frame_name = datumaro_item.id.split(self.VIDEO_FRAME_SEP)
-                    metadata_item = MetadataItemEntity(
-                        data=VideoMetadata(
-                            video_id=video_name,
-                            frame_idx=int(frame_name.split("_")[-1]),
-                            is_empty_frame=is_empty_frame,
-                        ),
-                    )
-                    dataset_item = DatasetItemEntity(
-                        image,
-                        self._get_ann_scene_entity(shapes),
-                        subset=subset,
-                        metadata=[metadata_item],
-                    )
-                    dataset_items.append(dataset_item)
-
-        found = [i for i, entity in enumerate(self.label_entities) if entity.name == self.EMPTY_FRAME_LABEL_NAME]
-        if found:
-            self.label_entities.pop(found[0])
-
-        return DatasetEntity(items=dataset_items)
+                        if self.label_entities[annotation.label].name == self.EMPTY_FRAME_LABEL_NAME:
+                            is_empty_frame = True
+                    video_name, frame_idx = datumaro_item.id.split(self.VIDEO_FRAME_SEP)
+                    datumaro_item.attributes['video_id'] = video_name
+                    datumaro_item.attributes['frame_idx'] = int(frame_idx.split('_')[-1])
```

Comment:
Why is this different to that of ActionClassification? 
```
datumaro_item.attributes['frame_idx'] = int(frame_idx)
```

Reply:
I do not precisely remember the reason. It's because we need to use this kind of variation since we use the AVA dataset for the ActionDetection.